src/renderer/components/input/input.tsx

```diff
@@ -90,7 +90,7 @@ export class Input extends React.Component<InputProps, State> {
     return this.state.valid;
   }
 
-  setValue(value: string) {
+  setValue(value = "") {
```

Comment:
Why do we change logic of `Input`?

Reply:
Because `<Wizard>` wraps its contents in a `<form>` and `<EditableList>` also uses the equivalent of `onEnter`, these changes are needed to make sure that the `<form>.onSubmit` isn't always called.